### PR TITLE
add OMSimulator package in install dir

### PR DESCRIPTION
**Pull request by arun3688**
### Purpose

This PR builds the OMSimulator package in install directory

### CODE REVIEW

File: src/OMSimulatorPython/CMakeLists.txt [hunk 1]
```diff
@@ -3,19 +3,27 @@ project(OMSimulatorPython)
...
+install(FILES "${CMAKE_CURRENT_BINARY_DIR}/OMSimulator.py" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+install(FILES "${CMAKE_INSTALL_PREFIX}/bin/${OMSIMULATORLIB_STRING}" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+install(FILES "${CMAKE_INSTALL_PREFIX}/bin/${OMSIMULATOR_TLM_STRING}" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
+
 IF (WIN32 AND MSVC)
   install(FILES OMSimulatorPython3.bat DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
 ELSEIF (WIN32 AND MINGW)
   install(FILES OMSimulatorPython3.bat DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
   install(FILES OMSimulatorPython3 DESTINATION ${CMAKE_INSTALL_PREFIX}/bin PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
+  install(FILES "${CMAKE_INSTALL_PREFIX}/bin/libomtlmsimulator.dll" DESTINATION ${CMAKE_INSTALL_PREFIX}/OMSimulator)
```
Comment (lochel) [inline src/OMSimulatorPython/CMakeLists.txt:26]: This should be removed, right?

**Review by lochel — CHANGES_REQUESTED** [review]
Where do you copy __init__.py?

File: src/OMSimulatorPython/CMakeLists.txt (hunk 1 above)
Comment (arun3688) [reply src/OMSimulatorPython/CMakeLists.txt]: removed https://github.com/OpenModelica/OMSimulator/pull/795/commits/5c162140aa719d488a73e5b698295444fc64e648

**Review by lochel — APPROVED** [review]
Weird to approve my own changes... 😕

**Pull request merged**